diffs.txt

```diff
@@ -0,0 +1,342 @@
+diff --git a/ksql-common/src/main/java/io/confluent/ksql/function/AggregateFunctionInitArguments.java b/ksql-common/src/main/java/io/confluent/ksql/function/AggregateFunctionInitArguments.java
```

Comment:
I dont think you meant to check this in?

Reply:
indeed! good catch ;)